index.html

```diff
@@ -216,6 +216,55 @@ <h2>Skills</h2>
                     </div>
                 </div>
             </section>
+            <!------------------------------------Contact section------------------------------------>
+            <section class="contact-form">
+                <div class="contact-header">
+                    <h1>I'm always interested in hearing about new projects, so if you'd 
+                    like to chat please get in touch</h1>
+                </div>
+                <div class="form-container">
+                    <form class="contact">
+                        <input type="text" placeholder="Full name" maxlength="30" required>
+                        <input type="email" placeholder="Email adderss" required>
+                        <textarea type="textarea" maxlength="500" placeholder="Hey I'd like ot connect to..."></textarea>
```

Comment:
- [ ] I have noticed that the form can be submitted when the `textarea` is empty. However, the project requirements suggest that all input fields must have the required attribute to activate the validation on submission. Therefore, I suggest that you consider validating the text area field for proper form submission.

Reply:
Dear @AlphaNtihinduka 
Thanks for your comment I the required attribute